src/main/java/com/cloudbees/jenkins/support/impl/RunDirectoryComponent.java

```diff
@@ -98,7 +98,7 @@ public boolean understandsSymlink() {
     @NonNull
     @Override
     public ComponentCategory getCategory() {
-        return ComponentCategory.PLATFORM;
+        return ComponentCategory.BUILDS;
```

Comment:
`category` not used in `ObjectComponent`s AFAICT, so this has no visible effect.